printer/printer.go

```diff
@@ -93,4 +103,16 @@ func PrintRepoReport(allEvents []types.AnnotatedEvent, runReport *types.RunRepor
 		fmt.Println()
 
 	}
+
```

Comment:
In order to keep with the spirit of making it really easy to understand exactly what happened with each run, I decided to treat pull requests and draft pull requests as two separate entities for reporting purposes. 

The result is more explicit messaging in the run report about which PRs were drafts and which draft PRs couldn't be opened due to repo incompatibility. If you don't pass the `--draft` flag, you'll get pull requests, and if you do pass it, you end up with draft pull requests.